optimize/util/optimize-commons/pom.xml

```diff
@@ -77,25 +77,31 @@
       <artifactId>json-path</artifactId>
       <version>${jsonpath.version}</version>
     </dependency>
+    <!-- unused/declared
     <dependency>
       <groupId>org.yaml</groupId>
       <artifactId>snakeyaml</artifactId>
       <version>2.3</version>
     </dependency>
```

Comment:
pretty sure that this was added to fix a CVE in the past. If this does not expose us (we could check via Snyk/trivy), then happy to remove

Reply:
I just tried to run Snyk and I got this output with and without that dependency:
```
~/workspace/camunda/optimize/util/optimize-commons (optimize-dependencies-bis) » snyk test       mattia.buccarella@Mattias-MacBook-Pro

Testing /Users/mattia.buccarella/workspace/camunda/optimize/util/optimize-commons...

Organization:      buccarel
Package manager:   maven
Target file:       pom.xml
Project name:      io.camunda.optimize:optimize-commons
Open source:       no
Project path:      /Users/mattia.buccarella/workspace/camunda/optimize/util/optimize-commons
Licenses:          enabled

✔ Tested 105 dependencies for known issues, no vulnerable paths found.

Next steps:
- Run `snyk monitor` to be notified about new related vulnerabilities.
- Run `snyk test` as part of your CI/test.


~/workspace/camunda/optimize/util/optimize-commons (optimize-dependencies-bis) »
```
Even the number of tested dependencies stays the same at 105.

Reply:
ok nice, perhaps the upstream dependency (I can't remember what it was) finally updated to a patched version of snakeyaml. In that case, this is probably good to rip out